vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
+
+| TYPE | REQUIRED | PLATFORM |
+|:--:|:--|:--|
+| boolean | Yes | Windows, macOS |
+
+[React Native Web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-9fe3ab194c72e9837a8ffc3e7c44f33d) introduced  `hovered` on `Pressable` to enable common mouse scenarios. In order to align and make this complete the work is as follows:
+
+- Windows has added `hovered` and marked `onMouseEnter/Leave` as to be deprecated [issue TBD]()
+- Add `hovered` to `Pressable` [issue TBD]()
+- macOS has added 'hovered' and marked their View component implementation of `onMouseEnter/Leave` as deprecated [issue TBD]()
+
+### `hovered` example
+
+``` js
+<Pressable style={({ hovered, pressed, focused }) => {
+          console.log(focused);
+          let backgroundColor = 'white';
+          if (hovered) {
+            backgroundColor = 'lightgray';
+          }
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
```

Comment:
Would be great if we could document semantics of how we handle overlapping components, since there has been changes/confusion to our own semantics there (see https://github.com/microsoft/react-native-windows/issues/4274 and https://github.com/microsoft/react-native-windows/issues/5901)

Reply:
@kikisaints See discussion on #5901. Am I correct in interpreting your WinUI animation as "mouse is 'inside' only the top-most element"? What is considered top-most? If I had `<Button><Image /><Text /></Button>` and mouse over the image, is the cursor no longer in/hovering the Button?

Philosophically, I'd argue that we should prefer doing things like web unless there is a compelling reason not to (say, direction from RN itself). Especially when the APIs here directly correspond to the web Pointer Events specs.